From 394f50b87ccbadbcd29975fa0742b0e3557f94ee Mon Sep 17 00:00:00 2001
From: Marek Vasut <marex@denx.de>
Date: Fri, 27 Mar 2020 02:58:37 +0100
Subject: [PATCH 60/97] ARM: dts: stm32: Add alternate pinmux for SAI2 pins

Add new mux option for SAI2 pins, this is used on AV96 board.

Signed-off-by: Marek Vasut <marex@denx.de>
Cc: Alexandre Torgue <alexandre.torgue@st.com>
Cc: Maxime Coquelin <mcoquelin.stm32@gmail.com>
Cc: Patrice Chotard <patrice.chotard@st.com>
Cc: Patrick Delaunay <patrick.delaunay@st.com>
Cc: linux-stm32@st-md-mailman.stormreply.com
To: linux-arm-kernel@lists.infradead.org
---
 arch/arm/boot/dts/stm32mp157-pinctrl.dtsi | 19 +++++++++++++++++++
 1 file changed, 19 insertions(+)

diff --git a/arch/arm/boot/dts/stm32mp157-pinctrl.dtsi b/arch/arm/boot/dts/stm32mp157-pinctrl.dtsi
index 00b03ce0befa8..5a38a82586e0e 100644
--- a/arch/arm/boot/dts/stm32mp157-pinctrl.dtsi
+++ b/arch/arm/boot/dts/stm32mp157-pinctrl.dtsi
@@ -815,6 +815,25 @@
 				};
 			};
 
+			sai2a_pins_c: sai2a-4 {
+				pins {
+					pinmux = <STM32_PINMUX('D', 13, AF10)>, /* SAI2_SCK_A */
+						 <STM32_PINMUX('D', 11, AF10)>, /* SAI2_SD_A */
+						 <STM32_PINMUX('D', 12, AF10)>; /* SAI2_FS_A */
+					slew-rate = <0>;
+					drive-push-pull;
+					bias-disable;
+				};
+			};
+
+			sai2a_sleep_pins_c: sai2a-5 {
+				pins {
+					pinmux = <STM32_PINMUX('D', 13, ANALOG)>, /* SAI2_SCK_A */
+						 <STM32_PINMUX('D', 11, ANALOG)>, /* SAI2_SD_A */
+						 <STM32_PINMUX('D', 12, ANALOG)>; /* SAI2_FS_A */
+				};
+			};
+
 			sai2b_pins_a: sai2b-0 {
 				pins1 {
 					pinmux = <STM32_PINMUX('E', 12, AF10)>, /* SAI2_SCK_B */
-- 
2.27.0

